Makefile

```diff
@@ -60,66 +60,21 @@ test-e2e: export OASIS_INDEXER_E2E = true
 test-e2e:
 	@$(GO) test -race -coverpkg=./... -coverprofile=coverage.txt -covermode=atomic -v ./tests/e2e
 
-fill-cache-for-e2e-regression:
-	@$(MAKE) fill-cache-for-e2e-regression-eden
-	@$(MAKE) fill-cache-for-e2e-regression-damask
+# To run specific suites, do e.g.
+# make E2E_REGRESSION_SUITES='suite1 suite2' test-e2e-regression
+E2E_REGRESSION_SUITES := eden damask
```

Comment:
ah so that's how you do it; i was trying to pass shell env vars to `make` before, with no luck.

Reply:
huh, but I though I read that env vars work too